backend/src/packages/chat-messages/chat-message.repository.ts

```diff
@@ -60,7 +60,10 @@ class ChatMessageRepository implements Repository {
   }
 
   public delete(): ReturnType<Repository['delete']> {
-    return Promise.resolve(true);
+    //TODO
+    const deletedId = 0;
```

Comment:
Is it mocked?

Reply:
Or, i just missed this repository to change

Reply:
So I returned it back to  return number in repository, so it is needed everywhere now. You can see this also here #307